.travis.yml

```diff
@@ -2,14 +2,16 @@ language: node_js
 sudo: required
 node_js:
   - "0.10"
+  - "4"
   - "4.4.3"
+  - "6"
+  - "8"
 services:
   - docker
 before_install:
   - sudo apt-get update
   - sudo apt-get install --assume-yes apache2-utils
   - npm install -g npm@2.13.5
```

Comment:
Should we still be using npm 2.x, even when running on node 6/8?

Reply:
I think that's a fair question. Perhaps @wtrocki can confirm if there's a requirement from some users?